app/Cancellation/OrderCancellation.php

```diff
@@ -46,8 +46,15 @@ public static function make(Order $order, $attendees): OrderCancellation
      */
     public function cancel(): void
     {
+        $orderAwaitingPayment = false;
+        if ($this->order->order_status_id == config('attendize.order.awaiting_payment')) {
```

Comment:
```suggestion
        if ($this->order->isAwaitingPayment()) {
```

Please extract this part into the `App\Models\Order` model? Then this piece of code can become a bit simpler.

You could drop `$orderAwaitingPayment` as a variable and use the order model method again, which reads a bit easier. 🙏